src/libecalc/common/stream.py

```diff
@@ -0,0 +1,26 @@
+from typing import Literal
+
+from libecalc.common.string_utils import to_camel_case
+from libecalc.common.utils.rates import TimeSeriesFloat, TimeSeriesRate
+from pydantic import BaseModel, Extra
+
+
+class StreamCondition(BaseModel):
```

Comment:
I guess usage here is in consumer system (only), but the entity should be called `Stream`, and it is the `Stream` that should be used in consumer system imo

Reply:
Only consumer system now, but I'm hoping/expecting this to be used for all new components, also in models if we want to introduce TimeSeries there..

It's used in compressor (and when this is approved I will add it to pump) so when we add a pump and compressor v2 (outside system) it will be used there also